packages/components/psammead-episode-list/src/index.test.jsx

```diff
@@ -3,36 +3,61 @@ import { render } from '@testing-library/react';
 import * as scripts from '@bbc/gel-foundations/scripts';
 import '@testing-library/jest-dom/extend-expect';
 
-import { renderEpisodes, exampleEpisodes } from './fixtures';
+import {
+  renderEpisodes,
+  renderVideoEpisodes,
+  exampleEpisodes,
+  exampleVideoEpisodes,
+} from './fixtures';
 
 describe('Episode List ', () => {
   it('should render the list', () => {
     const { getByRole } = render(
-      renderEpisodes(exampleEpisodes, scripts.latin, 'news', 'ltr'),
+      renderEpisodes({
+        episodes: exampleEpisodes,
+        script: scripts.latin,
+        service: 'news',
+        dir: 'ltr',
```

Comment:
Update `renderEpisodes` to use these settings by default, so they only need provided if they're being overridden